playbooks/openshift-management/private/config.yml

```diff
@@ -13,6 +13,16 @@
           status: "In Progress"
           start: "{{ lookup('pipe', 'date +%Y%m%d%H%M%SZ') }}"
 
+- name: Enable sebool container_manage_cgroup
+  hosts: nodes
+  gather_facts: false
+  tasks:
+  - name: Setting sebool container_manage_cgroup
```

Comment:
This should be moved to `roles/openshift_management/tasks/main.yml` so that it would be applied on the target hosts and executed during updates

Reply:
Oh, I see. If it should run on nodes it should use `oo_nodes_to_config` group, so that it would be working in AWX.

I'd suggest moving this to a separate playbook (and call it in `playbooks/openshift-management/private/config.yml`) so that this playbook could be called during upgrade too.

Reply:
@vrutkovs I have made the changes you asked for